### PR TITLE
Support pwsh preview error feedback providers

### LINKED ISSUE #170484: Add quick fix provider for new pwsh preview feedback providers
https://github.com/PowerShell/PowerShell/releases/tag/v7.4.0-preview.1

https://github.com/PowerShell/PowerShell/pull/18252

image

### PR DESCRIPTION
**Pull request by Tyriar**
Fixes #170484

I ended up adding both always, it can result in a lot of suggestions but it felt bad not suggesting some of them. We've talked about allow filtering that list before which would be cool here.

image

### CODE REVIEW

**Comment by Tyriar** [comment]
@daxian-dbw a quick review of how this is being matched would be appreciated 🙂

File: src/vs/workbench/contrib/terminal/test/browser/quickFixAddon.test.ts [hunk 1]
```diff
@@ -333,6 +333,102 @@ suite('QuickFixAddon', () => {
 			});
 		});
 	});
+	suite.only('pwsh feedback providers', () => {
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/test/browser/quickFixAddon.test.ts:336]:
```suggestion
	suite('pwsh feedback providers', () => {
```

**Comment by daxian-dbw** [comment]
The matching looks good to me. I just realize that you can receive the standard output from an EXE without redirecting the stdout of the EXE, which is very handy 😄 
About the fuzzy matching, the algorithm being used in PowerShell is not ideal, so the results are not very relevant sometimes.

**Comment by Tyriar** [comment]
@daxian-dbw I noticed some not so great results like `python -> pwsh`, since we're deferring to you we'll pick up improvements just by updating pwsh.

**Review by meganrogge — DISMISSED** [review]
Just remove the .only and LGTM

**Comment by daxian-dbw** [comment]
> I noticed some not so great results like `python -> pwsh`

Yeah, that's why it's remains an experimental feature (PSCommandNotFoundSuggestion).

**Review by meganrogge — APPROVED** [review]

**Pull request merged**